### DISCUSSION TITLE
Addition of Minshawy Reciter

### DISCUSSION THREAD

**Discussion by mahmoud-rashwan** [AlfaazPlus/QuranApp#471] [Ideas]
I absolutely love the app , thanks for your attentiveness to detail . Would you kindly add Minshawy as Reciter . For Me he is my personal favorite for his soothing and calming voice , same goes for my family . This would be much appreciated

**Comment by faisalcodes** [comment]
Thanks for the suggestion. The requested reciter is now available in the app.

**Reply by mahmoud-rashwan** [reply]
Many Thanks Faisal , much appreciated , Jazak Allah khayran :) , have a great day.
